src/core/core_messages_scheme.json

```diff
@@ -163,7 +157,16 @@
     "required": ["input"],
     "additionalProperties": false
   },
-  "GetPTTRequest": {},
+  "GetPTTRequest": {
+    "properties":{
+      "input": {
+        "type": "array",
+        "items": {"type": "string"}
```

Comment:
That's not how `GetPTTRequest` looks like.
GetPTTRequest contains `input`(as usual), and then it's an object with two parts `request` and `workerSig` both being strings.

where did you get that array from?
https://github.com/enigmampc/enigma-core/blob/develop/enigma-core/app/src/networking/ipc_listener.rs#L288
https://github.com/enigmampc/enigma-core/blob/develop/enigma-core/app/src/networking/messages.rs#L51

Reply:
Yes, this is how `GetPTTRequest` looks like, because this is the initial request that `P2P` makes to `core` and that array is the optional list of contract addresses. You are getting confused with `core` 's response to that initial request, and your links above specify core's response, but the request is actually specified here:

https://github.com/enigmampc/enigma-core/blob/5f005cdcbb4473954d97362899e1367e0ba887a6/enigma-core/app/src/networking/messages.rs#L115

Reply:
I'm sorry, I was wrong here. I've mistaken the GetPTTRequest request with the response. my mistake.